MonoGame.Framework/Graphics/Effect/EffectParameterCollection.cs

```diff
@@ -7,10 +7,21 @@ public class EffectParameterCollection : IEnumerable<EffectParameter>
         internal static readonly EffectParameterCollection Empty = new EffectParameterCollection(new EffectParameter[0]);
 
         private readonly EffectParameter[] _parameters;
+        private readonly Dictionary<string, int> _lookupDictionary = new Dictionary<string, int>();

```

Comment:
Do you mind changing this to `_indexLookup` or something that doesn't have `Dictionary` in it?

Reply:
np.